item-spec/common-metadata.md

```diff
@@ -34,12 +34,13 @@ Descriptive fields to give a basic overview of a STAC entity (e.g. Catalog, Coll
-| title       | string    | A human readable title describing the STAC entity.                                                                                                            |
-| description | string    | Detailed multi-line description to fully explain the STAC entity. [CommonMark 0.29](https://commonmark.org/) syntax MAY be used for rich text representation. |
-| keywords    | \[string] | List of keywords describing the STAC entity.                                                                                                                  |
-| roles       | \[string] | The semantic roles of the entity, e.g. for assets, links, providers, bands, etc.                                                                              |
+| Field Name  | Type      | Description                                                                                                                                                                                                                                                                     |
+| ----------- | --------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| title       | string    | A human readable title describing the STAC entity.                                                                                                                                                                                                                              |
+| description | string    | Detailed multi-line description to fully explain the STAC entity. [CommonMark 0.29](https://commonmark.org/) syntax MAY be used for rich text representation.                                                                                                                   |
+| keywords    | \[string] | List of keywords describing the STAC entity.                                                                                                                                                                                                                                    |
+| roles       | \[string] | The semantic roles of the entity, e.g. for assets, links, providers, bands, etc.                                                                                                                                                                                                |
+| attribution | string    | The attribution should be short and intended for presentation to a user, for example, in a corner of a map. Parts of the text can be links to other resources if additional information is needed. [CommonMark 0.29](https://commonmark.org/) syntax MAY be used for rich text. |
```

Comment:
In OGC API - Common it looks more like HTML is the markup language instead of CommonMark, although their language is a bit ambiguous.

Reply:
Since we allow markdown everywhere else when rich text is possible, allowing html seems inconsistent. md to html converter are pretty frequent

Reply:
Yeah, on the other hand it would better align with OGC and Markdown renderers can usually just leverage HTML, too.

Reply:
I hesitated but if you say so, I put HTML

Reply:
https://github.com/opengeospatial/ogcapi-common/issues/303#issuecomment-2098313243

Reply:
Funny how quick my "what should go wrong" yesterday was made obsolete. The HTML/Markup issue makes me think we might want to go for an extension...

Reply:
Add to discussion for next STAC meetup